pygmt/src/select.py

```diff
@@ -90,13 +90,13 @@ def select(data=None, outfile=None, **kwargs):
         - **c** select records NOT inside any point's circle of influence.
         - **f** select records NOT inside any of the polygons.
         - **g** will pass records inside the cells with z equal zero of the
-          grid mask in **gridmask**.
+          grid mask in ``gridmask``.
```

Comment:
grdmask is a GMT's module, so I think we should use 
```
:gmt-docs:`grdmask.html`
```

Reply:
I thought `gridmask` (with `i`) is the parameter (`-G` flag), as in the [upstream GMT documentation](https://docs.generic-mapping-tools.org/dev/gmtselect.html#g).

Reply:
You're right. I missed that part.